workflows/epigenetics/chipseq-pe/chipseq-pe.ga

```diff
@@ -1,6 +1,6 @@
 {
     "a_galaxy_workflow": "true",
-    "annotation": "Complete ChIP-seq analysis for paired-end sequencing data. Processes raw FASTQ files through adapter removal (cutadapt), alignment to reference genome (Bowtie2), and stringent quality filtering (MAPQ &gt;= 30, concordant pairs only). Peak calling with MACS2 optimized for paired-end reads identifies protein-DNA binding sites. Generates alignment files, peak calls, and quality metrics for downstream analysis.",
+    "annotation": "Complete ChIP-seq analysis for paired-end sequencing data. It goes from fastqs to peaks and coverage.",
```

Comment:
Was there a reason for removing the description ? I I think the adapter trimming, alignment and peak calling should be mentioned.